server/db/mysql/adapter.go

```diff
@@ -45,8 +46,11 @@ const (
 )
 
 type configType struct {
-	DSN    string `json:"dsn,omitempty"`
-	DBName string `json:"database,omitempty"`
+	DSN                    string `json:"dsn,omitempty"`
```

Comment:
Some connection parameters are specified in the DSN, some as explicit variables. Maybe get rid of the DSN altogether and use just the explicit variables? For example, construct DSN using https://pkg.go.dev/github.com/go-sql-driver/mysql#Config.FormatDSN

I don't insist. It just seems more consistent.

Reply:
These new params appear to be mysql driver specific settings.
The MySql specific DSN config does not contain them: https://pkg.go.dev/github.com/go-sql-driver/mysql#Config

Reply:
One thing we can do though is define a field of type `Config` and format it into DSN.
What do you think?

```go
type configType struct {
	DSNConfig Config `dsn_config`
        DBName string `json:"database,omitempty"`
        // Connection pool params.
	MaxOpenConns           int    `json:"max_open_conns,omitempty"`
	MaxIdleConns           int    `json:"max_idle_conns,omitempty"`
	ConnMaxLifetimeSeconds int    `json:"conn_max_lifetime_seconds,omitempty"`
}
```

Reply:
How about just pulling the vars out of `Config`:
```go
type configType struct {
	ms.Config
        // not needed anymore: DBName string `json:"database,omitempty"`
        // Connection pool params.
	MaxOpenConns           int    `json:"max_open_conns,omitempty"`
	MaxIdleConns           int    `json:"max_idle_conns,omitempty"`
	ConnMaxLifetimeSeconds int    `json:"conn_max_lifetime_seconds,omitempty"`
}
```

Reply:
Done. I've deprecated `dsn` and `database` fields.